frontend/tests/features/crudCollection.test.ts

```diff
@@ -0,0 +1,42 @@
+import { TEST_ENV, TEST_URL } from "tests/common/constants";
+import { goToPage } from "tests/utils/helpers";
+import { getText } from "tests/utils/selectors";
+
+const describeIfCalledByDevEnv = TEST_ENV === "dev" ? describe : describe.skip;
+
+describeIfCalledByDevEnv("CRUD Collection", async () => {
+  it("creates and deletes a collection", async () => {
+    await goToPage(TEST_URL);
+    // LOG IN
+    await page.click(getText("Log In"));
```

Comment:
do we need to type email and password?

Also it'd be great to extract the `login()` function as a utility function for reusability like we have in Explorer 🙏 

Thank you!